app/domain/authentication/authenticate.rb

```diff
@@ -28,7 +28,8 @@ def call
       validate_origin
       validate_credentials
       audit_success
-      new_token
+      token = new_token
+      @encoded_response ? Base64.strict_encode64(token.to_json) : token
```

Comment:
how about moving this logic inside `new_token`?

Reply:
Done